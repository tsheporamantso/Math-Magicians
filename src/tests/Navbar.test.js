import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navigation from '../page/Navbar';

describe('Navigation', () => {
  test('should match Navbar snapshot', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Navigation />
        ,
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
