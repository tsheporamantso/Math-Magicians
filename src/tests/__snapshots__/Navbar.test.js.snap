// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should match Navbar snapshot 1`] = `
Array [
  <nav
    className="nav"
  >
    <h1>
       Math Magicians
    </h1>
    <ul
      className="nav-items"
    >
      <li>
        <a
          className="links"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          className="links"
          href="/container"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li
        id="quotelink"
        style={
          Object {
            "border": "none",
          }
        }
      >
        <a
          className="links"
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>,
  ",",
]
`;
